src/libs/actions/Task.ts

```diff
@@ -295,7 +296,9 @@ function createTaskAndNavigate(
 
     API.write(WRITE_COMMANDS.CREATE_TASK, parameters, {optimisticData, successData, failureData});
 
-    Navigation.dismissModal(parentReportID);
+    if (shouldNavigate) {
```

Comment:
Consider to prevent clearOutTaskInfo function to avoid ONYX action

Reply:
Can you clear it out? I am not understanding this one.

Reply:
If we create task via mardown, in createTaskAndNavigate we don't need to call clearOutTaskInfo function 

https://github.com/Expensify/App/blob/86360745431edb355c91e5594c741a54e5f88581/src/libs/actions/Task.ts#L280

Reply:
Agree